doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -1279,7 +1292,255 @@ Default settings for logging are the following:
 - logging level: INFO;
 - rolling policy: whenever file size reaches 100 MB.
 
-# 17. Migrating to Remote Database Host
+# 17 Management REST API
+
+Central server has a REST API that can be used to do all the same server configuration operations that can be done
+using the web UI.
+
+Management REST API is protected with an API key based authentication. To execute REST calls, API keys need to be created.
+
+REST API is protected by TLS. Since server uses self-signed certificate, the caller needs to accept this (for example
+with `curl` you might use `--insecure` or `-k` option).
+
+Requests sent to REST API have a *limit for maximum size*. If a too large request is sent
+to REST API, it will not be processed, and http status 413 Payload too large will be returned.
+There is a different limit for binary file uploads, and for other requests.
+
+Limits are
+- 10MB for file uploads
+- 50KB for other requests
```

Comment:
Have these values been checked for the Central Server? These are the values for the Security Server, but the Central Server values might be different. For example:

https://nordic-institute.atlassian.net/browse/XRDDEV-2391?focusedCommentId=16549

Reply:
Actually, the Central Server doesn't have these limits currently. However, they will be added in this ticket and the default values will be the same that the Security Server uses. Therefore, these values are OK and no changes are needed here.

https://nordic-institute.atlassian.net/browse/XRDDEV-2436